package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "code-oss-dev",
   "version": "1.94.0",
-  "distro": "fcaeb73de7ac6ff11a3e732c63987e01b88341e7",
+  "distro": "75c0f563d85bdbeabcf0d76a4c6a0a64fdf9eb7b",
```

Comment:
This isn't the latest on distro FYI. It does seem to only move forward so I'll approve though.

Reply:
Actually the one after this seems to be a revert. Checking this in might be a bad idea. I will close this.